### PR TITLE
[feature] Export Networks for NeTEx France

### PR DESCRIPTION
**Pull request by woshilapin**

### CODE REVIEW

File: src/netex_france/exporter.rs [hunk 1]
```diff
@@ -24,16 +29,19 @@ use std::{
...
     Stops,
+    Lines,
 }
 
 impl Display for VersionType {
     fn fmt(&self, fmt: &mut Formatter) -> std::result::Result<(), fmt::Error> {
         use VersionType::*;
         match self {
             Stops => write!(fmt, "ARRET"),
+            Lines => write!(fmt, "LINE"),
```
Comment (ArnaudOggy) [inline src/netex_france/exporter.rs]: It should be "LIGNE", in french (specs). No ?

Comment (woshilapin) [reply src/netex_france/exporter.rs]: You're right, nice catch. Fixed in 01dd9729c2ef7bf03c17d93407cdf679a00dcb5e.

File: tests/fixtures/netex_france/lignes.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<PublicationDelivery version="1.09:FR-NETEX_LIGNE-2.1-1.0" xmlns="http://www.netex.org.uk/netex" xmlns:core="http://www.govtalk.gov.uk/core" xmlns:gml="http://www.opengis.net/gml/3.2" xmlns:ifopt="http://www.ifopt.org.uk/ifopt" xmlns:siri="http://www.siri.org.uk/siri" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.netex.org.uk/netex">
+	<PublicationTimestamp>2019-04-03T17:19:00</PublicationTimestamp>
+	<ParticipantRef>Participant</ParticipantRef>
+	<dataObjects>
+		<CompositeFrame>
```
Comment (ArnaudOggy) [inline tests/fixtures/netex_france/lignes.xml]: CompositeFrame needs an id and a version

File: tests/fixtures/netex_france/lignes.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+			<frames version="any">
+				<ServiceFrame version="any">
```
Comment (ArnaudOggy) [inline tests/fixtures/netex_france/lignes.xml]: ServiceFrame needs an id

File: tests/fixtures/netex_france/lignes.xml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<PublicationDelivery version="1.09:FR-NETEX_LIGNE-2.1-1.0" xmlns="http://www.netex.org.uk/netex" xmlns:core="http://www.govtalk.gov.uk/core" xmlns:gml="http://www.opengis.net/gml/3.2" xmlns:ifopt="http://www.ifopt.org.uk/ifopt" xmlns:siri="http://www.siri.org.uk/siri" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.netex.org.uk/netex">
+	<PublicationTimestamp>2019-04-03T17:19:00</PublicationTimestamp>
+	<ParticipantRef>Participant</ParticipantRef>
+	<dataObjects>
+		<CompositeFrame>
+			<frames version="any">
```
Comment (ArnaudOggy) [inline tests/fixtures/netex_france/lignes.xml]: frames version doesn't exist (can remove it)

File: tests/fixtures/netex_france/lignes.xml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+				<ServiceFrame version="any">
+					<Network id="FR:network:TGN:" version="any">
+						<Name>The Great Network</Name>
+						<LineRef ref="FR:line:M1">
```
Comment (ArnaudOggy) [inline tests/fixtures/netex_france/lignes.xml]: Networks needs a members section to contain LineRef sections

File: tests/fixtures/netex_france/lignes.xml (hunk 4 above)
Comment (woshilapin) [reply tests/fixtures/netex_france/lignes.xml]: Done in 538c47ae29253286e751395f9afd92a8f38a91e5.

File: tests/fixtures/netex_france/lignes.xml (hunk 2 above)
Comment (woshilapin) [reply tests/fixtures/netex_france/lignes.xml]: Done in c34cd9bccb8c601766b32e5d31ce13be3885c513.

File: tests/fixtures/netex_france/lignes.xml (hunk 3 above)
Comment (woshilapin) [reply tests/fixtures/netex_france/lignes.xml]: Done in c34cd9bccb8c601766b32e5d31ce13be3885c513.

File: tests/fixtures/netex_france/lignes.xml (hunk 5 above)
Comment (woshilapin) [reply tests/fixtures/netex_france/lignes.xml]: Done in d335dfc16a98add274bbd87ae20d07722f2d7838.

**Pull request merged**